Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Scripts/UpdateSliderTrackLine.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+public class UpdateSliderTrackLine : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject activeLine = null;
+    private float roundedValue;
+
+    public void OnSliderUpdated(SliderEventData eventData)
+    {
+            roundedValue = Mathf.Round(eventData.NewValue * 100f) / 100f;
+            activeLine.transform.localScale = new Vector3 (transform.localScale.x, roundedValue, transform.localScale.z);
```

Comment:
It feels a little weird to see two different classes which are basically completely the same except for the axis on which the change is applied - it feels like there should just be one MonoBehaviour with another dropdown/flag which controls whether or not the X or Y axis (or z axis!) is what is actually moved along.

Reply:
I've taken out the rounded value and am just using eventData.NewValue to drive the y-axis scale of the track. Ideally this could be expanded to work for the X and Z axes, but I think that can happen in the next iteration.